### PR TITLE
367 GitHub actions

### PR DESCRIPTION
**Pull request by newtewt**
# Description

Using github actions to build the project now and disable travis. I don't have access to disable the check for travis here. 

medic/medic-conf#367

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or integration tests where appropriate
- Backwards compatible: Works with existing data and configuration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by newtewt** [comment]
Ok, after reviewing the documentation and a bit of experimenting. This will show 2 builds but also shows the individual matrix so it looks like 6. This is how our travis flow works now except a little less noise.

**Comment by garethbowen** [comment]
I've removed the travis requirement from this repo and will add the actions requirement once this is merged.

File: .github/workflows/build.yml
Comment (garethbowen) [inline .github/workflows/build.yml:36]: What is this env var used for?

**Comment by newtewt** [comment]
Thanks @garethbowen just need a review and I can merge.

File: .travis.yml
Comment (garethbowen) [inline .travis.yml:10]: Dropping this line is dangerous - if anyone accidentally commits a `.only` then only that unit test will run. This happens more than you'd think.

It _may_ be covered by our eslint rules.

It'd be worth making sure it's caught somewhere. If it's really hard to implement in GA then raise an issue for it to unblock this PR.

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice work! Thanks. I've put a couple of comments inline.

**Comment by garethbowen** [comment]
I take it @emailfornc is a secondary account or something?

File: .travis.yml
Comment (newtewt) [reply .travis.yml:10]: I'm not setting the  `--forbid-ony` flag when running mocha. If there are any `.only` it will fail.

**Comment by newtewt** [comment]
> I take it @emailfornc is a secondary account or something?

yeah it was a second account I used a while back

File: .github/workflows/build.yml
Comment (newtewt) [reply .github/workflows/build.yml]: I have removed it. Was from the sample config and I didn't remove.

**Review by garethbowen — APPROVED** [review]
Great work! Ready to merge. Thanks.

**Pull request merged**
